src/qml/pages/node/Shutdown.qml

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../../controls"
+
+Page {
+    background: null
+
+    ColumnLayout {
+        anchors.centerIn: parent
+        width: parent.width
+        spacing: 10
+        Button {
+            Layout.alignment: Qt.AlignCenter
+            Layout.bottomMargin: 20
+            background: null
+            icon.source: "image://images/shutdown"
+            icon.color: Theme.color.neutral9
+            icon.width: 60
+            icon.height: 60
+        }
+        Header {
+            Layout.alignment: Qt.AlignCenter
+            Layout.fillWidth: true
+            Layout.leftMargin: 20
+            Layout.rightMargin: 20
+            header: qsTr("Saving...")
+            headerBold: true
+        }
+        Header {
```

Comment:
actually it can just be the following 😊 : 

``` diff
diff --git a/src/qml/pages/node/Shutdown.qml b/src/qml/pages/node/Shutdown.qml
index 26600f8ce..3141e4046 100644
--- a/src/qml/pages/node/Shutdown.qml
+++ b/src/qml/pages/node/Shutdown.qml
@@ -30,15 +30,8 @@ Page {
             Layout.rightMargin: 20
             header: qsTr("Saving...")
             headerBold: true
-        }
-        Header {
-            Layout.alignment: Qt.AlignCenter
-            Layout.fillWidth: true
-            Layout.leftMargin: 20
-            Layout.rightMargin: 20
-            header: qsTr("Do not shut down the computer until this is done.")
-            headerSize: 18
-            headerColor: Theme.color.neutral7
+            description: qsTr("Do not shut down the computer until this is done.")
+            descriptionColor: Theme.color.neutral7
         }
     }
 }
```

Reply:
oh yeah, this makes more sense